scripts/install/nltk_dictionaries.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -eo pipefail
+
+echo "MACHINE LEARNING WORKER NOTE: Next, you will be asked to install some wordlists for the natural language processing parts of Augur.  Please be advised that the machine learning workers, including the clustering worker, discourse worker, messsage_analysis_worker, pull_request_analysis_worker, and the LSTM model of the insight_worker will not function fully without these lists."
+
+read -r -p "Would you like to install required NLTK word lists for machine learning workers? [Y/n] " response
+case "$response" in
+  [yY][eE][sS]|[yY])
+    echo "Installing..."
+    mkdir ~/nltk_wordlist_installations_
+    mkdir /usr/local/share/nltk-wordlists
+    python -m nltk.downloader stopwords -d /usr/local/share >> ~/nltk_wordlist_installations/stopwords.log
+    cp /usr/local/share/stopwords /usr/local/share/nltk-wordlists
+    python -m nltk.downloader punkt -d /usr/local/share  >> ~/nltk_wordlist_installations/punkt.log               
+    cp /usr/local/share/punkt /usr/local/share/nltk-wordlists
+    python -m nltk.downloader popular -d /usr/local/share >> ~/nltk_wordlist_installations/popular.log
+    cp /usr/local/share/popular /usr/local/share/nltk-wordlists
+    python -m nltk.downloader universal_tagset -d /usr/local/share >> ~/nltk_wordlist_installations/universal_tagset.log
+    cp /usr/local/share/universal_tagset /usr/local/share/nltk-wordlists
+    echo "Done!"
+    echo "Logs are stored in your home directory under '/usr/local/share, /usr/local/share/nltk-worklists, and in some instances based on operating system, your home directroy'."
```

Comment:
Okay, there are still problems with this section.
1. Like I mentioned last time, they should not be going into `.log` files. Those have a very specific connatation and most programs will not hestitate to delete log files if needed, as they are generally viewed as temporary. **These files do not fit this profile, we need to make sure they are persisted and there is little chance of them being deleted on accident. Please change the file extension.**
2. Storing things in `/usr/local/` is usually fine, but not when each word gets its own folder, to me that is a waste of space. On top of that, there is already a `$HOME/.augur/` folder to store Augur-specific data. I should have recommended this explicitly last time; please store the stopwords in a folder under `$HOME/.augur/` (also, use `$HOME` instead of `~`, it is more standardized across platforms). 99% of the time, the user will have the correct access to write to this location, and the 1% of times they don't is not really an issue right now as that case is EXCEEDINGLY rare.
3. Instead of writing to output files, then copying them to a new location, just write them to the output location directly. Like: `python -m nltk.downloader universal_tagset -d $HOME/.augur/nltk-wordlists/`. This will write the output folder directly to `$HOME/.augur/nltk-wordlists/`, meaning we can skip the redundant copy step. This also lets us be consistent across platforms.

Reply:
@ccarterlandis : The only thing I will point out is that 
1. I don't need to copy them to their own directory.  That was more so I could keep track of what I installed
2. NLTK data sets / word sets, have a strict regimine of where they will be found in the path. 
 - The users' home directory, under `nltk_data` or one of these options: 
 -  /usr/share/nltk_data',
 - /usr/local/share/nltk_data
 - /usr/lib/nltk_data
 - /usr/local/lib/nltk_data

So, of those, which do you prefer. Remember that many people may share these datasets for reasons other than Augur, and so that's why the NLTK default does not include an application directory.